sysutils/beats8/src/opnsense/mvc/app/models/OPNsense/Beats8/Filebeat.xml

```diff
@@ -0,0 +1,55 @@
+<model>
+    <mount>//OPNsense/filebeat</mount>
+    <description>
+        Send logs to elasticsearch
+    </description>
+    <items>
+        <enabled type="BooleanField">
+            <Default>0</Default>
+            <Required>Y</Required>
+        </enabled>
+        <modules>
+            <enabled type="OptionField">
+                <OptionValues>
+                    <suricata>Suricata (Intrusion Detection)</suricata>
+                </OptionValues>
+                <Multiple>Y</Multiple>
+            </enabled>
+        </modules>
+        <inputs>
+            <enabled type="OptionField">
+                <Default>audit</Default>
+                <OptionValues>
+                    <audit>Audit</audit>
+                    <configd>Backend</configd>
+                    <boot>Boot</boot>
+                    <system>General</system>
+                    <lighttpd>Web GUI</lighttpd>
+                </OptionValues>
+                <Multiple>Y</Multiple>
+            </enabled>
+        </inputs>
+        <output>
+            <elasticsearch>
+                <hosts type="TextField">
```

Comment:
might consider a `HostnameField` for validation

Reply:
It might not support everything, but constraining input is usually a good starting point (and difficult to fix later). When both address and port need to be specified, you could also take a look at `IPPortField`

Reply:
Perfect, in that case I'll opt for an `UrlField`, which provides a constraint while supporting most setups. Simplified configurations like just a hostname (e.g., `elastic`) will need to be provided as an URL (e.g., `http://elastic:9200/`).